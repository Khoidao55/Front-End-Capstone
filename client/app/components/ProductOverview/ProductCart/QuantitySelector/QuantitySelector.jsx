import React from 'react';

const QuantitySelector = ({ selectedSize, sizeQuantity, arraySkus }) => {
  if (arraySkus !== undefined) {
    const arrayOfSize = [];
    for (let i = 1; i <= sizeQuantity; i += 1) {
      arrayOfSize.push(i);
    }
    switch (selectedSize) {
      default:
        return (
          <div>
            <select>
              <option>Select A Quantity</option>
            </select>
          </div>
        );
      case selectedSize:
        return (
          <div>
            <select>
              <option>Select A Quantity</option>
              {arrayOfSize.map((quantity) => {
                return (
                  <option key={quantity} value={quantity}>{quantity}</option>
                )
              })}
            </select>
          </div>
        );
    }
  }
};

export default QuantitySelector;
