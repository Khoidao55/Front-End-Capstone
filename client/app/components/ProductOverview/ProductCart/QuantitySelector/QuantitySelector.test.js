import React from 'react';
import { shallow } from 'enzyme';
import QuantitySelector from './QuantitySelector.jsx';

describe('Quantity Selector', () => {
  test('should render my component', () => {
    const wrapper = shallow(<QuantitySelector />);

    expect(wrapper).toMatchSnapshot();
  });
});
