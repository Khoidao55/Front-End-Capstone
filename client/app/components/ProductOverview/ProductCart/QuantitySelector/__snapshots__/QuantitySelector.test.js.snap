// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quantity Selector should render my component 1`] = `ShallowWrapper {}`;
